### PR TITLE
create a job to deploy mita

### PR DESCRIPTION
**Pull request by alfredodeza**
Requires manual addition of `JENKINS_PRADO` and `JENKINS_PRADO_TOKEN` on the Jenkins UI

### CODE REVIEW

**Comment by ktdreyer** [comment]
nice

File: mita-deploy/config/definitions/ceph-tag.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+- scm:
```
Comment (andrewschoen) [inline mita-deploy/config/definitions/ceph-tag.yml]: this file should be named ``mita-deploy.yml`` not ``ceph-tag.yml``

Comment (alfredodeza) [inline mita-deploy/config/definitions/ceph-tag.yml]: that is egregious

you don't like mix and matching names?

**Comment by alfredodeza** [comment]
test this please man I swear

**Pull request merged**